### ISSUE TITLE
PDF evaluation documentation from accessibility team

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#63383]
## About your team

- Team name: Accessibility Community of Practice
- OCTO-DE product owner: @artsymartha68 
- Accessibility specialist: @SarahKay8 

 
### Describe what you need in a few sentences:

I would like for @SarahKay8 to write down some doc about how she approaches a PDF audit request. It's not something we do a lot of but it would be nice to have the OCTO process written down. 

I don't think we need full documentation of how to make a PDF accessible, there is plenty of that in the world. Just the process for evaluating and determining what it needs, perhaps? What tools you use? Once you take a look at the PDF, how do you estimate the effort? Etc.

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [x] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?
N/A

#### Timing for your next step: 
N/A

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
N/A

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [x] No
 
  
### When do you expect to launch your product to 100% of users?
N/A
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [x] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

 
## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by SarahKay8** [comment]
https://docs.google.com/document/d/12jPFHeaKgjEpHGtKIDuPsArUhi0E56c5_9z_mGNZ4pU/edit Hi @artsymartha68 attached is my PDF documentation which is also shared via email. Could you let me know if I should add more?

**Comment by artsymartha68** [comment]
SK this is so great! So much great info.

As someone who uses InDesign from time to time, can you maybe add a link to your favorite doc for how to set up an InDD file to output an accessible PDF (or as well as possible)? 

Also: can we move this doc to Confluence so everyone can collaborate more easily?

**Comment by SarahKay8** [comment]
Will do! @artsymartha68 yes this can be on confluence 🌼

**Comment by SarahKay8** [comment]
@artsymartha68 I added the inDesign Section!

**Comment by SarahKay8** [comment]
Closing! :) @artsymartha68

**Comment by sara-amanda** [comment]
## Post-Ticket Closure, Related Activity
**January 12, 2024:** @coforma-jamie completed the following related ticket as part of our a11y documentation efforts: [a11y Documentation] Create PDF creation/remediation page based on existing documentation #73623 - Thank you, Jamie!
